src/components/dashboard/Claim.js

```diff
@@ -166,8 +166,11 @@ const Claim = props => {
     }
   }
 
-  const faceRecognition = () => {
-    screenProps.push('FRIntro', { from: 'Claim' })
+  const faceRecognition = async () => {
+    await goodWallet.isCitizen().then(_ => gdstore.set('isLoggedInCitizen')(_))
```

Comment:
why do we need this? whats the edge case that gdstore isn't set correctly previously?